src/science/mathematics/big-integers.md

```diff
@@ -0,0 +1,30 @@
+# Miscellaneous
+
+## Big integers
+
+[![num-badge]][num] [![cat-science-badge]][cat-science]
+
+If we need to make calculations on integers exceeding size of standard integer type, we can use BigInt.
```

Comment:
Prefer talking about the code rather than the author or audience.  "Calculation for integers exceeding 128 bits are possible with BigInt.

Also use backticks and brackets (including a link to the API).